sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1101,6 +1101,10 @@ class Analyzer(
           result
         case UnresolvedExtractValue(child, fieldExpr) if child.resolved =>
           ExtractValue(child, fieldExpr, resolver)
+        case f @ UnresolvedFunction(_, children, _, Some(filter)) =>
```

Comment:
do we still need this change if we do https://github.com/apache/spark/pull/26420/files#r347947991 ?

Reply:
No need.